docs/blog.md

```diff
@@ -25,7 +25,9 @@ Then, of course, there are users who use the keyboard heavily or exclusively. Th
  
 In any case, the `tab` key allows users to navigate sequentially through interactive elements on the page (such as form fields) with just a keyboard. The _only way_ for keyboard users to know which element is active (and thus how they might potentially interact with it via the keyboard) is via a visual indictor of what thing is focused - i.e., by default, the __focus ring__).
 
-Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, since designs vary, what the particular focus ring should look like could vary too. On a white page background, a light blue outline might be really appropriate - but if your interface _is_ light blue, it might be necessary to consider this part of branding and perhaps provide something like:
+Given this, it's tempting to put our foot down and ignore the subtleties of the issue.  We could simply say, "when a thing has focus it *must* get the focus ring indicator". However, this runs into issues before we even consider user context.
```

Comment:
Well, it seems like we are kind of making the case that styling focus actually isn't incompletely solved - that it's problematic.  Maybe "However this quickly runs into issues, even in really basic cases"